src/components/ReviewCard.jsx

```diff
@@ -1,14 +1,21 @@
 import "../styles/ReviewCard.css"
+import { FullReview } from "./FullReview"
+import {Link} from "react-router-dom"
 
-export const ReviewCard = ({title, category, image, date, comments, votes}) => {
+export const ReviewCard = ({title, category, image, date, comments, votes, review_id}) => {
 
     return(
-        <div className="review-card">
-            <h3 className="card-title">{title}</h3>
-            <p>Category: {category}</p>
-            <img src={image} alt="review-thumbnail" className="review-thumbnail"/>
-            <p className="review-info"><strong>Date Posted:</strong> {date.slice(0, 10)} <strong>Votes:</strong> {votes} <strong>Comments:</strong> {comments}</p>
-            <button className="view-full-review-button">Read Full Review</button>
-        </div>
+        <main>
+            <div className="review-card">
+                <h3 className="card-title">{title}</h3>
+                <p>Category: {category}</p>
+                <img src={image} alt="review-thumbnail" className="review-thumbnail"/>
+                <p className="review-info"><strong>Date Posted:</strong> {date.slice(0, 10)} <strong>Votes:</strong> {votes} <strong>Comments:</strong> {comments}</p>
+                <Link to={`/reviews/${review_id}`}
+                className="view-full-review-button">
+                    <button className="view-full-review-button-text" ><strong>Read Full Review</strong></button>
```

Comment:
Here this button doesn't have much functionality. You are still making use of the <Link> tag, which is fine. Please note that instead of having a button, you could what the title, or even the image, title and description into the <Link> tag to make it clickable. The was you did it is fine, this is just a suggestion for the future.